src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/RemoveAzureContainerGroupCommand.cs

```diff
@@ -0,0 +1,51 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ContainerInstance.Models;
+using Microsoft.Azure.Management.ContainerInstance;
+
+namespace Microsoft.Azure.Commands.ContainerInstance
+{
+    /// <summary>
+    /// Remove-AzureRmContainerGroup
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, ContainerGroupNoun, SupportsShouldProcess = true), OutputType(typeof(PSContainerGroup))]
+    public class RemoveAzureContainerGroupCommand : ContainerInstanceCmdletBase
+    {
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource group name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The container group name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        public override void ExecuteCmdlet()
```

Comment:
@yolo3301 please add a parameter set that has one parameter, `InputObject` of type `PSContainerGroup`, that has the property `ValueFromPipeline = true` so a user can pipe the result from `Get-AzureRmContainerGroup` to this cmdlet.

Also, please add another parameter set that has one parameter, `ResourceId` of type `string`, that has the property `ValueFromPipeByPropertyName = true` so a user can pipe the result from the generic cmdlets to this cmdlet.

You can view more information on piping in PowerShell [here](https://github.com/Azure/azure-powershell/wiki/Piping-in-PowerShell).

Reply:
Added all